app/api/items/path_from_root.go

```diff
@@ -17,24 +17,31 @@ import (
 //	description: >
 //		Finds a path from any of root items to a given item.
 //
-//
 //		The path consists only of the items visible to the participant
 //		(`can_view`>='content' for all the items except for the last one and `can_view`>='info' for the last one).
-//		Of all possible paths the service chooses the one having missing/not-started results located closer
-//		to the end of the path, preferring paths having less missing/not-started results and having higher values of `attempt_id`.
-//		The chain of attempts of the path cannot have missing results for items requiring explicit entry or not started results
-//		within or below ended/not-allowing-submissions attempts.
 //
+//		Of all possible paths, the service chooses the one having:
+//			* missing/not-started results located closer to the end of the path,
+//			* preferring paths having less missing/not-started results,
+//			* and having higher values of `attempt_id`.
 //
-//		If `as_team_id` is given, the attempts/results of the path are linked to the `as_team_id` group instead of the user's self group.
+//		For a path to be returned, each of its items must:
+//			* Either have `require_explicit_entry`=0 ,
+//			* Or if it has `require_explicit_entry=1`,
+//				then the following condition must be fulfilled, except if it is the last item of the path:
+//				the item must have at least one result with `started`=1 AND its attempt must have
+//					(`NOW()` < `attempt.ended_at` ) OR (`NOW()` < `attempt.allows_submissions_until`)).
```

Comment:
Shouldn't it be a "AND" between the 2 conditions?

Reply:
fixed, actually it was wrong. I updated it with `(`attempt.ended_at` IS NULL) AND (`NOW()` < `attempt.allows_submissions_until`)`, which is what the query is doing.